### ISSUE TITLE
[BUG] Error Connecting to Server

### ISSUE THREAD

**Issue by mapas01** [nixcamic/Monobus#3]
When I hit search, I get an error "The remote server returned an error: (403) Forbidden.

**Comment by nixcamic** [comment]
I don't get this error, can you access getcomics.org in your browser?

**Comment by mapas01** [comment]
> I don't get this error, can you access getcomics.org in your browser?

Yes I can get to the site on my browser. I thought maybe it was because I was running it via a network drive, but I put the files locally and still same message.  Windows 10, in case that matters.  Also tried it on another machine, same message.

**Comment by kiranwayne** [comment]
I have the same issue . It is likely you can get around the issue by using a VPN or by using mobile data.

**Background and additional information:**

- I had a similar issue with Omnibus on release 1.5.1. It was working perfectly for a long time, and suddenly Search wouldn't display any results, and, behind-the-scenes, the http response was a 403. Nothing has changed on my end

- I noticed the issue happened only when I was on my ISP network (Verizon Fios NE USA) but not when I was on a VPN or when I used mobile data
 
- I worked with fireshaper, the author of Omnibus, and it was found the User Agent string was not populated properly in some areas. Fixing that addressed the issue, until now

**Current status and observations:**

- When on Fios and not on a VPN, both Omnibus and this fork (Monobus) return a 403 - Omnibus does so silently and Monobus displays the error on the screen
 
- When on a VPN, both work fine
 
- Leaving the application open and disconnecting the VPN - Omnibus returns a 403 whereas Monobus continues to work fine until the application is closed and opened again

This leads me to wonder if this has something to do with cookies. A cursory glance at the code in form1.cs did not help me understand why Omnibus and Monobus would behave differently.
